src/lib/simulation/simulatedKillables.ts

```diff
@@ -120,6 +123,21 @@ export const simulatedKillables: SimulatedKillable[] = [
 			return loot;
 		}
 	},
+	{
+		name: 'King Goldemar',
+		isCustom: true,
+		message: '**Assumptions**:\n- Solo\n- Ring of Luck equipped',
+		loot: (quantity: number): Bank => {
+			const loot = new Bank();
+			for (let i = 0; i < quantity; i++) {
+				if (roll(calcDwwhChance(1, true))) {
+					loot.add('Broken dwarven warhammer');
+				}
+				loot.add(KingGoldemarLootTable.roll());
```

Comment:
This should be `roll(quantity)` outside the loop, its a lot more efficient/faster as it doesnt need to create an object for every single iteration. If you must do it in a loop (not in this case here), the other way is: `.roll(1, {targetBank: loot})`

Reply:
It doesn't necessarily need to be inside the loop, I was just going off how the others are structured. Let me know if this change looks good and or you want me change the others aswell.